src/main/java/net/imagej/ops/threshold/binaryMask/DefaultBinaryMask.java

```diff
@@ -0,0 +1,108 @@
+package net.imagej.ops.threshold.binaryMask;
+
+import net.imagej.ops.Contingent;
+import net.imagej.ops.Ops;
+import net.imagej.ops.special.hybrid.AbstractBinaryHybridCF;
+import net.imglib2.Cursor;
+import net.imglib2.RandomAccess;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.img.Img;
+import net.imglib2.img.array.ArrayImgs;
+import net.imglib2.type.NativeType;
+import net.imglib2.type.logic.BitType;
+import net.imglib2.type.numeric.RealType;
+import org.scijava.plugin.Plugin;
+
+import java.util.stream.IntStream;
+
+/**
+ * An op that creates a binary mask of the elements within the given thresholds
+ *
+ * @author Richard Domander (Royal Veterinary College, London)
+ */
+@Plugin(type = Ops.Threshold.BinaryMask.class)
+public class DefaultBinaryMask<T extends RealType<T> & NativeType<T>>
+        extends AbstractBinaryHybridCF<RandomAccessibleInterval<T>, DefaultBinaryMask.Thresholds<T>, Img<BitType>>
+        implements Ops.Threshold.BinaryMask, Contingent {
+    @Override
+    public void compute2(final RandomAccessibleInterval<T> interval, final Thresholds<T> thresholds,
+            final Img<BitType> output) {
+        final Cursor<BitType> cursor = output.localizingCursor();
+        final RandomAccess<T> intervalAccess = interval.randomAccess();
+        final long[] maskPosition = new long[3];
+        final long[] intervalPosition = new long[interval.numDimensions()];
+
+        while (cursor.hasNext()) {
+            cursor.fwd();
+            cursor.localize(maskPosition);
+            System.arraycopy(maskPosition, 0, intervalPosition, 0, 3);
+            intervalAccess.setPosition(intervalPosition);
+            final T intervalElement = intervalAccess.get();
+            if (intervalElement.compareTo(thresholds.min) >= 0 && intervalElement.compareTo(thresholds.max) <= 0) {
+                cursor.get().setOne();
+            }
+        }
+    }
+
+    @Override
+    public Img<BitType> createOutput(final RandomAccessibleInterval<T> interval, final Thresholds<T> thresholds) {
+        final long[] dimensions = new long[interval.numDimensions()];
+        interval.dimensions(dimensions);
+        return ArrayImgs.bits(dimensions);
```

Comment:
also I would initialize the `create` using `Functions.create(...)`. However, @ctrueden will change the way how one creates `Function`s and `Computer`s and `Hybrid`s.